packages/reakit/src/Toolbar/__examples__/ToolbarWithPopover/__tests__/index-test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { render, press } from "reakit-test-utils";
+import ToolbarWithPopover from "..";
+
+const popoverText =
+  "This trip lasts three days and takes place in the otherworldly environment of Ares Station.";
+
+test("renders toolbar items with closed popover", () => {
+  const { getByText: text } = render(<ToolbarWithPopover />);
+
+  expect(text("Beach")).toBeVisible();
+  expect(text("Mountain")).toBeVisible();
+  expect(text("Mars")).toBeVisible();
+
+  expect(text(popoverText)).not.toBeVisible();
+});
+
+test("allows to navigate toolbar items through keyboard", () => {
+  const { getByText: text } = render(<ToolbarWithPopover />);
+
+  press.Tab();
+  expect(text("Beach")).toHaveFocus();
+
+  press.ArrowRight();
+  expect(text("Mountain")).toHaveFocus();
+
+  press.ArrowLeft();
+  expect(text("Beach")).toHaveFocus();
+
+  press.ArrowRight();
+  press.ArrowRight();
+  expect(text("Mars")).toHaveFocus();
```

Comment:
We can test here if we can move out of `Mars` using `ArrowLeft`. Right now it's not working.